aiosqlite/core.py

```diff
@@ -258,11 +258,11 @@ def in_transaction(self) -> bool:
         return self._conn.in_transaction
 
     @property
-    def isolation_level(self) -> str:
+    def isolation_level(self) -> Optional[str]:
         return self._conn.isolation_level
 
     @isolation_level.setter
-    def isolation_level(self, value: str) -> None:
+    def isolation_level(self, value: Optional[str]) -> None:
```

Comment:
Let's return these to the original annotation you suggested. Maybe make that a separate alias like:

```py
IsolationLevel = Optional[Literal["DEFERRED", "IMMEDIATE", "EXCLUSIVE"]]
```

So then you can use that like `value: IsolationLevel`